source/_integrations/sun.markdown

```diff
@@ -18,22 +19,28 @@ below the horizon. The sun can be used within automation as
 [sun_trigger]: /docs/automation/trigger/#sun-trigger
 [sun_condition]: /docs/scripts/conditions/#sun-condition
 
-## Configuration
+## Configured by default
 
-This integration is by default enabled, unless you've disabled or removed the [`default_config:`](/integrations/default_config/) line from your configuration. If that is the case, the following example shows you how to enable this integration manually:
+This integration is by default configured and installed, and you don't need
+to configure it yourself, unless you've disabled or removed the
+[`default_config:`](/integrations/default_config/) line from your
+YAML configuration.
+
+If that is the case, you can configure it as described in the next paragraphs.
+
+{% include integrations/config_flow.md %}
+
+## YAML Configuration
+
+Alternatlively, this integration can be configured and set up manually via YAML
+as well. To enable the sun integration in your installation, add the
+following to your `configuration.yaml` file:
 
 ```yaml
 # Example configuration.yaml entry
 sun:
 ```
 
-{% configuration %}
-elevation:
-  description: "The (physical) elevation of your location, in meters above sea level. Defaults to the `elevation` in `configuration.yaml`, which is retrieved from Google Maps if not set."
-  required: false
-  type: integer
```

Comment:
Oops, this was still documented 🤦

Should we add a note about how to actually configure location and elevation though?

Reply:
This is part of the onboarding processes and UI core configuration for a long time already. You actually can't skip it.

Reply:
Added my link to introduction text